app/controllers/webhooks/stripe/invoices_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Webhooks::Stripe::InvoicesController < Webhooks::BaseController
+  skip_before_action :verify_authenticity_token, only: [:create, :charge_event]
+  before_action :validate_event_type, only: [:create, :charge_event] 
+
+  def create
+    event = get_event(params)
+    invoice_handler = InvoiceHandler.new(event)
+    invoice_handler.create_invoice!
+
+    head :ok, status: 202
```

Comment:
[Stripe says](https://stripe.com/docs/webhooks#configuring-your-webhook-settings) we should be returning a 200.

![image](https://cloud.githubusercontent.com/assets/15848772/19838189/6ac392a4-9ea0-11e6-8136-664d39d99c7f.png)